### ISSUE TITLE
There is no method to sort a grid

### ISSUE THREAD

**Issue by Artur-** [vaadin/vaadin-grid-flow#314]
In Framework 8 there is
```
    public void sort(Column<T, ?> column) {
    public void sort(Column<T, ?> column, SortDirection direction) {
    public void sort(String columnId) {
    public void sort(String columnId, SortDirection direction) {
    public void clearSortOrder() {
    public void setSortOrder(List<GridSortOrder<T>> order) {
    public void setSortOrder(GridSortOrderBuilder<T> builder) {
    public List<GridSortOrder<T>> getSortOrder() {

```
Here, there is nothing?

**Comment by Legioth** [comment]
Related to and/or duplicate of https://github.com/vaadin/vaadin-grid-flow/issues/13

**Comment by Artur-** [comment]
and #200  which is marked as fixed through #290
